src/inference/src/os/lin/lin_system_conf.cpp

```diff
@@ -202,6 +203,8 @@ CPU::CPU() {
                     numa_node_map.insert(std::pair<int, int>(numa_node_list[i][j], i * _numa_nodes / _sockets + j));
                 }
             }
+            _socketid_mapping_table.clear();
```

Comment:
If it is just reset table before using, line 206 and 207 are duplicated with 221 and 222. Please merge them together.